Add type annotations for torch._C._distributed_c10d module.

xuzhao9 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #46624 Add type annotations to torch._C._distributed_rpc module.
* **#46623 Add type annotations for torch._C._distributed_c10d module.**
* #46622 Add Python declaration of torch._C and torch._C._autograd modules.

Differential Revision: [D24761606](https://our.internmc.facebook.com/intern/diff/D24761606)

xuzhao9 committed:
Add type annotations for torch._C._distributed_c10d module.

[ghstack-poisoned]

xuzhao9 committed:
Update on "Add type annotations for torch._C._distributed_c10d module."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add type annotations for torch._C._distributed_c10d module."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add type annotations for torch._C._distributed_c10d module."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add type annotations for torch._C._distributed_c10d module."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add type annotations for torch._C._distributed_c10d module."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add type annotations for torch._C._distributed_c10d module."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add type annotations for torch._C._distributed_c10d module."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add type annotations for torch._C._distributed_c10d module."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add type annotations for torch._C._distributed_c10d module."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add type annotations for torch._C._distributed_c10d module."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add type annotations for torch._C._distributed_c10d module."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add type annotations for torch._C._distributed_c10d module."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add type annotations for torch._C._distributed_c10d module."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add type annotations for torch._C._distributed_c10d module."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add type annotations for torch._C._distributed_c10d module."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add type annotations for torch._C._distributed_c10d module."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add type annotations for torch._C._distributed_c10d module."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add type annotations for torch._C._distributed_c10d module."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add type annotations for torch._C._distributed_c10d module."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add type annotations for torch._C._distributed_c10d module."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add type annotations for torch._C._distributed_c10d module."




[ghstack-poisoned]

walterddr reviewed (approved):
looks good. please import to phab

xuzhao9 committed:
Update on "Add type annotations for torch._C._distributed_c10d module."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add type annotations for torch._C._distributed_c10d module."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add type annotations for torch._C._distributed_c10d module."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add type annotations for torch._C._distributed_c10d module."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add type annotations for torch._C._distributed_c10d module."


Differential Revision: [D24761606](https://our.internmc.facebook.com/intern/diff/D24761606)

[ghstack-poisoned]
